src/main/java/digital/slovensko/autogram/server/AssetsEndpoint.java

```diff
@@ -6,17 +6,13 @@
 import digital.slovensko.autogram.server.errors.InvalidUrlParamException;
 
 import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
 import java.util.List;
 
 import static java.util.Objects.requireNonNull;
 
 public class AssetsEndpoint implements HttpHandler {
     private static final List<String> assets;
-    private static final Path assetsPath;
     static  {
-        assetsPath = Path.of(requireNonNull(AssetsEndpoint.class.getResource("index.html")).getPath()).getParent().resolve("assets");
```

Comment:
Toto padalo ešte pri spustení appky. Keďže je to ale na takomto "static" mieste, exception sa nedal nikde chytiť a len to zobrazilo "Failed to load JVM".